pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -31,7 +31,6 @@ message Schema {
 	}
 
     required string name = 1;
-    required bytes version = 2;
```

Comment:
Shouldn't we need to know the version in some cases? 

I'd imagine that a producer deriving the schema from a class type, won't know the version. But a consumer receiving a schema, might need the version?